packages/amplify-graphql-schema-generator/src/schema-generator/generate-schema.ts

```diff
@@ -62,6 +64,20 @@ export const generateGraphQLSchema = (schema: Schema, existingSchemaDocument?: D
   return schemaStr;
 };
 
+const isEnum = (type: FieldType): boolean => {
```

Comment:
Yeah, I think if the response on this is `: type is EnumType` then you can clean up the calling code a bit.

Reply:
I'm not sure I follow. Since we passing all the fields to this method, casting to EnumType throwing typescript errors.